app/models/order.rb

```diff
@@ -2,6 +2,7 @@ class Order < ApplicationRecord
   has_many :order_items
   has_many :items, through: :order_items
   belongs_to :user
+  belongs_to :address, optional: true
```

Comment:
I'm curious why this has the optional: true. Wouldn't an order always need an address?